odxtools/diaglayerraw.py

```diff
@@ -163,7 +163,7 @@ def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) ->
         ]
 
         communication_parameters = [
-            CommunicationParameterRef.from_et(el, doc_frags, variant_type)
+            CommunicationParameterRef.from_et(el, doc_frags)
```

Comment:
variant_type is not used outside of here, maybe remove it?

Reply:
ok, I tried this, but *actually* it is used in a few places, e.g. https://github.com/mercedes-benz/odxtools/blob/main/odxtools/diaglayer.py#L589-L590

Reply:
I am talking about the local variable used only in this function (declared in line 74), not about all occurrences of `variant_type`

Reply:
That one is passed to `DiagLayerRaw` on line 181...